crates/ruff_python_ast/src/symbol_imports.rs

```diff
@@ -0,0 +1,57 @@
+use std::fmt;
+
+/// A representation of an individual symbol imported via any import statement.
```

Comment:
We have a different understanding of the term `symbol`. In my view, these are possible `declarations` of a symbol. I'll add a more in-depth explanation in #3777